x-pack/plugins/session_view/common/mock/app_root_provider.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, ReactNode, useMemo } from 'react';
+import { I18nProvider } from '@kbn/i18n/react';
+import { Router } from 'react-router-dom';
+import { History } from 'history';
+import useObservable from 'react-use/lib/useObservable';
+import { EuiThemeProvider } from '../../../../../src/plugins/kibana_react/common';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { KibanaContextProvider } from '../../../../../src/plugins/kibana_react/public';
+import { CoreStart } from '../../../../../src/core/public';
+
+/**
+ * Provides the context for rendering the endpoint app
+ */
+export const AppRootProvider = memo<{
```

Comment:
Should we combine this with `app_context_render`? since nothing else is importing this

Reply:
might be a good idea, I'll try that